### ISSUE TITLE
Add podman support

### ISSUE THREAD

**Issue by JediNite** [WDCommunity/wdpksrc#76]
Look into the option of adding podman support to the WD series of NAS instead of using docker.  Newer versions of docker don't work properly on arm, so this may be an alternative.

https://podman.io/getting-started/installation

Thoughts ?

**Comment by stefaang** [comment]
TBH, podman is new to me. But this seems like a good challenge to work on.
The caveat is that my gut feel says that docker could be way more mature than podman.
I may get access to an EX2Ultra soon to help out here.

**Comment by JediNite** [comment]
Yeah thinking it might be a good alternative to try and get working. No urgent rush on it and no guarantees we will get it to work.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
